assets/js/view/blogview.js

```diff
@@ -0,0 +1,34 @@
+// View for each individual item
+var BlogView = Backbone.View.extend({
+
+  template: AppTemplates.sidebar,
+
+  el: '#sidebar',
+
+  initialize: function() {
+    this.render();
+    this.listenTo(this.collection, 'sync', this.render);
+  },
+
+  render: function() {
+    var html = this.template(this.collection.toJSON());
+    this.$el.html(html);
+    return this;
+  },
+});
+
+// View for all
+var AppView = Backbone.View.extend({
+  template: AppTemplates.app,
+
+  el: '#target',
+
+  initialize: function() {
+    var _this = this;
+    var html = _this.template(_this.model.toJSON());
```

Comment:
Set this up in a render function. Also you don't need `_this` since you aren't fighting with any callbacks.

On top of that it looks like you are trying to call `this.model.toJSON()` but look what you pass to your `AppView` up here: https://github.com/santiagomarvin/6.4-backbone-exercises/pull/2/files#diff-36f8d737c0d72482b317e9b95c615dc2R7